File: Sweep/Data/Score.cs
using GolfSweep.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GolfSweep.Data
{
    public class Score : BaseEntity
    {
        public int Gross { get; set; }
        public int Nett { get; set; }
        public int Points { get; set; }
        public int HoleNumber { get; set; }
    }
}

File: Sweep/ClientApp/app/Score.ts
export interface Score {
    id: string;
    gross: number;
    nett: number;
    points: number;
    holeNumber: number;
    course: Course
}

export interface Course {
    id: string;
    name: string;
}

export interface CreateScoreCard {
    id: string;
    courseId: string;
    playerId: string;
}